bfd/elfxx-riscv.c

```diff
@@ -1770,6 +1770,40 @@ riscv_parse_add_implicit_subsets (riscv_parse_subset_t *rps)
 			      RISCV_UNKNOWN_VERSION,
 			      RISCV_UNKNOWN_VERSION, TRUE);
     }
+
+  if ((riscv_lookup_subset (rps->subset_list, "b", &subset)))
+    {
+      riscv_parse_add_subset (rps, "zba",
+                              RISCV_UNKNOWN_VERSION,
```

Comment:
Wrong GNU indents.

Reply:
Removed, sorry for that I will pay more attention with the indent problems.